src/components/Header/Header.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useMemo } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[],
```

Comment:
why do you need to pass the whole array of todos here, we can just pass the nr of completed and uncompleted todos